roles/common/tasks/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 # tasks file for common
+- name: Determinate repofile for target
+  ansible.builtin.include_tasks: determinate-repofile.yml
```

Comment:
Maybe we could set it only after the next validate task for systems succeeds? No point running it before.